src/components/UserCard.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div
+    class="user-card"
+    @click.stop
+    @mouseover="$emit('mouseover', $event)"
+    @mouseleave="$emit('mouseleave', $event)"
+  >
+    <UserInfo :address="address" />
+  </div>
+</template>
+
+<script>
+import UserInfo from './UserInfo.vue';
+
+export default {
+  components: {
+    UserInfo,
+  },
+  props: {
+    address: { type: String, required: true },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.user-card {
+  top: 15px;
+  width: 75%;
+  border: 1px solid $card_border_color;
+  border-radius: 0.5rem;
+  position: absolute;
+  z-index: 1;
+  overflow: hidden;
+  cursor: default;
+  box-shadow: 4px 4px 4px 0 rgba(0, 0, 0, 0.3);
+  background-color: $thumbnail_background_color;
+}
+
+::v-deep .profile__stats {
```

Comment:
Put this inside `.user-card`, otherwise, it looks like a mutation of other component's styles without any condition (like being inside `.user-card`).